File: src/main/java/com/ptah/repository/userprofiling/OrganizationNominationRepository.java
package com.ptah.repository.userprofiling;

import com.ptah.entity.userprofiling.OrganizationNomination;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface OrganizationNominationRepository extends JpaRepository<OrganizationNomination, Long> {
    List<OrganizationNomination> findByUserId(Long userId);
}

File: backend/src/main/kotlin/com/ptah/repository/userprofiling/OrganizationNominationRepository.kt
package com.ptah.repository.userprofiling

import com.ptah.entity.userprofiling.OrganizationNomination
import org.springframework.data.jpa.repository.JpaRepository

interface OrganizationNominationRepository : JpaRepository<OrganizationNomination?, Long?> {
    fun findByUserId(userId: Long?): List<OrganizationNomination?>
}